examples/slider/multithumb-slider.html

```diff
@@ -7,6 +7,7 @@
   <!--  Core js and css shared by all examples; do not modify when using this template. -->
   <link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
   <link rel="stylesheet" href="../css/core.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>

```

Comment:
Is there a way we can do this without jQuery?

Reply:
It would be a lot more obnoxious to get the javascript and css resources but certainly we could. Why should we not use jQuery?

Reply:
Feels weird to include an entire library to open an example in an external resource 🤷‍♀. Is it a one time annoyance? Or is it annoying every time we add an example?

Reply:
one time annoyance, to replace the `$.get` ajaxes in the `examples/js/examples.js` file.

Reply:
Can't say that I won't to write that code, lol :) but I hear your point.

Reply:
I can have a look